### PR TITLE
Updated hanksTodo android Readme.md 

### LINKED ISSUE #81: Update README.md file for HanksTodo android app
Readme.md file for hanksTodo android app needs to be updated with the latest changes. Need to have clear steps on how to build and run the app from scratch.

### PR DESCRIPTION
**Pull request by Vishwa4jeet**
Fixes #81 

Made changes to the readme file as per the latest code changes.

### CODE REVIEW

File: hanksTodo/README.md [hunk 1]
```diff
@@ -1,30 +1,70 @@
-# Hankstodo demo application
+# HanksTodo android demo application
 
-Follow the following steps to build and run hanksTodo demo app.
+### About the App
+The hanksTodo app allows users to create multiple todo lists. Users can maintain multiple todo items within a todo list. Once you are done with a todo item or a list, you can go ahead and delete them.
+
+### Amino MicroServices
+1. **TodoListManager** :
+    Second class in the app to implement the Microservice and makes use of the Default Deployment Manager(DM).
```
Comment (VenuReddyHuawei) [inline hanksTodo/README.md]: Make it - "This class implements"

File: hanksTodo/README.md [hunk 2]
```diff
@@ -1,30 +1,70 @@
...
+    
+2. **TodoList** :
+    This class implements the MicroService and makes use of multi-DM. The combination of DMs used is DHT and ConsensusRSM.
```
Comment (VenuReddyHuawei) [inline hanksTodo/README.md]: Rephrase it to -
This class implements the MicroService and uses a combination of DHT and ConsensusRSM DMs.

Comment (Vishwa4jeet) [reply hanksTodo/README.md]: Done.

File: hanksTodo/README.md (hunk 1 above)
Comment (Vishwa4jeet) [reply hanksTodo/README.md]: Done.
